components/prism-handlebars.js

```diff
@@ -21,8 +21,8 @@
 				variable: /[\s\S]+/
 			}
 		},
-		'punctuation': /[!"#%&'()*+,.\/;<=>@\[\\\]^`{|}~]/,
-		'variable': /[^!"#%&'()*+,.\/;<=>@\[\\\]^`{|}~\s]+/
+		'punctuation': /[!"#%&':()*+,\.\/;<=>@\[\\\]^`{|}~]/,
```

Comment:
You don't have to escape the dot.

```suggestion
		'punctuation': /[!"#%&':()*+,.\/;<=>@\[\\\]^`{|}~]/,
```